tests/yam/agama/patch_agama_tests.pm

```diff
@@ -13,7 +13,6 @@ use Utils::Architectures 'is_s390x';
 
 sub run {
     unless (is_s390x) {
-        assert_screen('agama-product-selection', 120);
         select_console 'root-console';
```

Comment:
then this one is better as one-liner:
`select_console 'root-console' unless is_s390x;`
but just an style comment :)

Reply:
Solved d5762697117fd642678a1d3cf122784b44c2fcf3